buildenv/jenkins/variables/defaults.yml

```diff
@@ -157,203 +159,209 @@ s390x_linux:
+    14: '--with-openssl=fetched'
     next: '--with-openssl=fetched'
   excluded_tests:
     11:
       ? special.system
-    12:
-      ? special.system
     13:
       ? special.system
+    14:
+      ? special.system
+    next:
+      ? special.system
 #========================================#
 # AIX PPC 64bits Compressed Pointers
 #========================================#
 ppc64_aix:
   boot_jdk:
     8: '/usr/java7'
     11: '/usr/java11_64'
-    12: '/usr/java11_64'
     13: '/usr/java12_64'
+    14: '/usr/java13_64'
     next: '/usr/java13_64'
   release:
     8: 'aix-ppc64-normal-server-release'
     11: 'aix-ppc64-normal-server-release'
-    12: 'aix-ppc64-server-release'
     13: 'aix-ppc64-server-release'
+    14: 'aix-ppc64-server-release'
     next: 'aix-ppc64-server-release'
   freemarker: '/home/jenkins/freemarker.jar'
   openjdk_reference_repo: '/home/jenkins/openjdk_cache'
   node_labels:
     build:
       8: 'ci.role.build && hw.arch.ppc64 && sw.os.aix'
       11: 'ci.role.build && hw.arch.ppc64 && sw.os.aix'
-      12: 'ci.role.build && hw.arch.ppc64 && sw.os.aix'
       13: 'ci.role.build && hw.arch.ppc64 && sw.os.aix'
+      14: 'ci.role.build && hw.arch.ppc64 && sw.os.aix'
       next: 'ci.role.build && hw.arch.ppc64 && sw.os.aix'
   extra_getsource_options:
     8: '--openssl-version=1.1.1a'
     11: '--openssl-version=1.1.1a'
-    12: '--openssl-version=1.1.1a'
     13: '--openssl-version=1.1.1a'
+    14: '--openssl-version=1.1.1a'
     next: '--openssl-version=1.1.1a'
   extra_configure_options:
     8: '--with-cups-include=/opt/freeware/include --disable-ccache --with-jobs=8 --with-openssl=fetched'
     11: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors --with-jobs=8 --with-openssl=fetched'
-    12: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors --with-jobs=8 --with-openssl=fetched'
     13: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors --with-jobs=8 --with-openssl=fetched'
+    14: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors --with-jobs=8 --with-openssl=fetched'
     next: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors --with-jobs=8 --with-openssl=fetched'
   build_env:
     vars:
       8: 'PATH+XLC=/opt/IBM/xlC/13.1.3/bin:/opt/IBM/xlc/13.1.3/bin'
       11: 'PATH+XLC=/opt/IBM/xlC/13.1.3/bin:/opt/IBM/xlc/13.1.3/bin'
-      12: 'PATH+XLC=/opt/IBM/xlC/13.1.3/bin:/opt/IBM/xlc/13.1.3/bin'
       13: 'PATH+XLC=/opt/IBM/xlC/16.1.0/bin:/opt/IBM/xlc/16.1.0/bin CC=xlclang CXX=xlclang++'
+      14: 'PATH+XLC=/opt/IBM/xlC/16.1.0/bin:/opt/IBM/xlc/16.1.0/bin CC=xlclang CXX=xlclang++'
       next: 'PATH+XLC=/opt/IBM/xlC/16.1.0/bin:/opt/IBM/xlc/16.1.0/bin CC=xlclang CXX=xlclang++'
   excluded_tests:
     8:
       ? special.system
     11:
       ? special.system
-    12:
-      ? special.system
     13:
       ? special.system
+    14:
+      ? special.system
+    next:
+      ? special.system
```

Comment:
We don't have the machine availability to handle these extra tests.  At least not until we have more AIX brought online.

Reply:
Or maybe this is irrelevant as we don't run these test mightly/daily.

Reply:
We don't generally run `next` testing iirc. I was just adding the exclude to be more consistent.